plugins/gear/gear.h

```diff
@@ -57,12 +56,25 @@ public slots:
 private:
 
     QPointF *center;
+    QDoubleSpinBox *rotateBox;
     QSpinBox *nteethBox;
-    QDoubleSpinBox *pitchBox;
+    QDoubleSpinBox *modulusBox;
     QDoubleSpinBox *pressureBox;
     QDoubleSpinBox *addendumBox;
     QDoubleSpinBox *dedendumBox;
-    QComboBox *typeBox;
```

Comment:
With the use of C++11 features, it's easy to initialize pointers inside class declaration.
For new code we'd like to see this:
```    QSpinBox  *n1Box {nullptr};```
Please use this initialization for all your new pointer members and add it to existing pointers too.